decidim-core/config/locales/en.yml

```diff
@@ -203,6 +203,7 @@ en:
     newsletter_mailer:
       newsletter:
         note: You received this email because you're subscribed to newsletters on %{organization_name}. You can change your settings on your <a href="%{link}">notifications page</a>.
+        see_on_website: Can’t see this email correctly? View on <a href="%{link}">website</a>. 
```

Comment:
Maybe change the final sentence to something like "View it on the website"? Note I added "it" and "the". Link is OK.